ssm/ssm2.py

```diff
@@ -116,8 +111,16 @@ def __init__(self, hosts_and_ports, qpath, cert, key, dest=None, listen=None,
                 if not crypto.verify_cert_path(self._enc_cert, self._capath, self._check_crls):
                     raise Ssm2Exception('Failed to verify server certificate %s against CA path %s.' 
                                          % (self._enc_cert, self._capath))
-            
-    
+
+        # If the overall SSM log level is info, we want to only
+        # see log entries from stomp.py at the warning level and above.
+        if logging.getLogger("ssm.ssm2").getEffectiveLevel() == logging.INFO:
+            logging.getLogger("stomp.py").setLevel(logging.WARNING)
+        # If the overall SSM log level is debug, we want to only
+        # see log entries from stomp.py at the info level and above.
```

Comment:
Is stomp.py's DEBUG output any different to its INFO output?

Reply:
Yes, in `DEBUG` mode `stomp.py` logs the message content sent, whereas in `INFO` it just logs the headers

Reply:
Ok, that sounds excessive for our purposes, so I think we can leave this as it is. Seems to best match what we had previously.